consensus/ising/proposerservice.go

```diff
@@ -94,28 +94,37 @@ func FilterNoderByIDs(nodes []protocol.Noder, nids []uint64) (ret []protocol.Nod
 }
 
 func (ps *ProposerService) ProcessRollback() {
-	step := BlockRollbackStep
-	forkingPointSet := false
-	for !forkingPointSet {
-		probeHeight := int(ps.forkCache.currentHeight) - step
-		if probeHeight <= 0 {
+	forkedPointFound := false
+	genesisChecked := false
+	ps.forkCache.probeHeight = ledger.DefaultLedger.Store.GetHeight()
+
+	for !forkedPointFound {
+		if genesisChecked && 0 == ps.forkCache.probeHeight {
```

Comment:
should be '&' ?

Reply:
&& is ok.

Reply:
@yilunzhang Yes. Have tested on nkn-testing join into Cluster 3. Deleted Chain dir and re-sync from height 0.